backend/app/mcp_server/server.py

```
@@ -0,0 +1,377 @@
+        mode=mode,
+    )
+    return json.dumps(result, ensure_ascii=False, default=str)
+
+
+# ============== Starlette App Factory ==============
+
+
+def _create_system_mcp_app() -> Starlette:
+    """Create Starlette app for system MCP server."""
+
+    async def health_check(request: Request):
```

Ruff (0.14.14)

Unused function argument: `request`

(ARG001)